src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class FigureSupplier {
+
+    public double getNum() {
+        return Math.random() * 5 + 1;
+    }
+
+    public Figures getRandomFigure() {
+        String randomColor = new ColorSupplier().getRandomColor();
+        double randomNumber = Math.random() * 4 + 1;
```

Comment:
```suggestion
        int randomNumber = (int) (Math.random() * 4 + 1);
```

let's make `randomNumber` as `int` from beginning, then you don't need to convert it every time in `if` block